compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -653,8 +653,14 @@ void addCPULinalgExtTileAndVectorizePipeline(
   }
 }
 
-void addCPUDefaultPassPipeline(OpPassManager &funcPassManager) {
+void addCPUDefaultPassPipeline(OpPassManager &funcPassManager,
+                               FailureOr<TilingConfig> &tilingConfig) {
   addTileAndDistributePasses(funcPassManager);
```

Comment:
It does not make sense to add these passes when the TilingConfig is not present, so perhaps we should just move it into the if-body?

Here is the context for other reviewers, there are some dispatches that do not have any TilingInterface ops, so we can't find any TilingConfig on any op. I think we still need the bufferization because of the flow/hal binding ops. This is one such examples: https://gist.github.com/pashu123/0b5329e8a60311b634ce9d2230381b20